internal/clients/http/ec2/ec2_client.go

```diff
@@ -55,7 +55,7 @@ func newAssumedEC2ClientWithRegion(ctx context.Context, arn string, region strin
 func getStsAssumedCredentials(ctx context.Context, arn string) (*stsTypes.Credentials, error) {
 	logger := ctxval.Logger(ctx)
 
-	cfg, err := awsCfg.LoadDefaultConfig(ctx, awsCfg.WithRegion(config.AWS.Region),
+	cfg, err := awsCfg.LoadDefaultConfig(ctx, awsCfg.WithRegion("us-east-1"),
```

Comment:
Wait why you hardcode it here when there is the region argument in the function declaration?

Reply:
Yeah, this was first take, sorry I assume, that this config can be on global region, because role assuming is region agnostic, but I'll use the region from argument for now :)

Reply:
fixed with todo, I believe there is room for improvement in the role assume.